src/components/AudioTrack/PlayPause.vue

```diff
@@ -65,11 +40,27 @@ export default {
     isPlaying() {
       return this.status === 'playing'
     },
+    /**
+     * Get the button label based on the current status of the player.
+     */
     label() {
-      return this.$t(getLabelFromStatus(this.status))
+      const statusVerbMap = {
+        playing: 'pause',
+        paused: 'play',
+        played: 'replay',
+      }
+      return `play-pause.${statusVerbMap[this.status]}`
     },
+    /**
+     * Get the button icon based on the current status of the player.
+     */
     icon() {
-      return STATUS_TO_ICON[this.status]
+      const statusIconMap = {
+        playing: pauseIcon,
+        paused: playIcon,
+        played: replayIcon,
+      }
+      return statusIconMap[this.status]
```

Comment:
Just curious, it's not a big perf cost or anything, but why put this (and the `statusVerbMap`) constants in the function body instead of on the module level? Inside the function body it'll be redefined every time the function runs and also redefined for each instance of the component rendering to the page.

It's not a big deal ultimately, they're small objects memory wise, but just curious if you prefer this for some reason over what existed before?

Reply:
I preferred this change because it meant keeping the code of the component inside the component itself. As you said it's not a performance hit so it made sense to me to keep it all inside the top level component `export`.

Reply:
Technically, moving them to `data` might make more sense. Updated in 3714540.

Reply:
Oh interesting, why does moving them to `data` make more sense? Wouldn't that turn them into reactive objects? I always thought it was weird that the only way to get data into templates in the options API, even static data, was to make it reactive 🤔